multi_gpu/env.yml

```diff
@@ -0,0 +1,10 @@
+channels:
+- pytorch-nightly
+- conda-forge
+- defaults
+dependencies:
+- pytorch
+- torchvision
+- torchaudio
+- cudatoolkit=11.3
+#- wandb ## Uncomment to install Weights and Biases for profiling.
```

Comment:
Should the readme mention why W&B appears in various files but is commented out?